src/topology/maps.lean

```diff
@@ -369,6 +369,56 @@ this ▸ continuous_iff_is_closed.mp h s hs
 
 end is_closed_map
 
+section open_embedding
+variables [topological_space α] [topological_space β] [topological_space γ]
+
+/-- A open embedding is an embedding with open image. -/
+def open_embedding (f : α → β) : Prop := embedding f ∧ is_open (range f)
```

Comment:
Given that we have `is_open_map`, I think `is_open_embedding` would be better. (I know it is impossible to have perfect coherence here, since we already have `embedding` and `continuous`, say -- but I think I would also prefer `is_embedding`, by the way).

Reply:
I copied `open_embedding` from the existing `closed_embedding` so for now I think this is the "most consistent" name. But we can certainly discuss what to do about `is_` prefixes in general.